### PR TITLE
fix pdf links for preprints (and add links for old ones)

### PR DESCRIPTION
**Pull request by micokoch**
Addresses #219 (and fixes other links, too).

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -175,6 +175,7 @@
   slug: gibson-mortality
   authors: Gibson GC, Reich NG, Sheldon D
   preprint: https://www.medrxiv.org/content/10.1101/2020.12.22.20248736v2
+  pdf: https://www.medrxiv.org/content/10.1101/2020.12.22.20248736v2.full.pdf
```
Comment (nickreich) [inline _data/publications.yml]: Could we make this PDF link to the published article PDF, not the preprint PDF?

**Review by nickreich — CHANGES_REQUESTED** [review]
Thanks for adding all the PDF links! See one place where maybe we could point to a journal page link? or if not because it's not open access, maybe we could put the PDF file somewhere on our website where it could be accessed? I think it should at some point be public on NCBI website.

File: _data/publications.yml (hunk 1 above)
Comment (micokoch) [reply _data/publications.yml]: Hi Nick. I sent a separate message about this, since I cannot access the published article PDF.

Comment (micokoch) [reply _data/publications.yml]: Thanks for the link - the article was updated.

**Pull request merged**